pkg/gateway/operations/listobjects.go

```diff
@@ -442,6 +442,7 @@ func handleListMultipartUploads(w http.ResponseWriter, req *http.Request, o *Rep
 		opts.KeyMarker = &keyMarker
 	}
 	mpuResp, err := o.BlockStore.ListMultipartUploads(req.Context(), block.ObjectPointer{
+		StorageID:        o.Repository.StorageID,
```

Comment:
How could TestListMultipartUploads pass without this?  I think perhaps still using blank StorageIDs in those tests?

Reply:
AFAICT code was introduced [here](https://github.com/treeverse/lakeFS/pull/8522#diff-e6688d58729bf0fd6aacdec37eb6b9818620532d385b5b028dd291f12e038f30R49) with no tests.

Reply:
The issue is with adding untested code to the gateway.  I am more comfortable with the new code as long as there are no storage IDs anywhere - I guess we can pull this code untested.  But not sure of the benefit TBH.